src/app/cheat-sheets/game-base/space-age/space-age.component.html

```diff
@@ -154,7 +159,7 @@ <h5>Planet Start</h5>
         <app-factorio-icon [icon]="dataService.getFactorioIcon('Recycler')"></app-factorio-icon>
         <app-factorio-icon [icon]="dataService.getFactorioIcon('Fast_inserter')"></app-factorio-icon>
         <app-factorio-icon [icon]="dataService.getFactorioIcon('Requester_chest')"></app-factorio-icon>
-        to process recycling
+        to process recycling when using a logistics network.
```

Comment:
Do we still need this bit of text? It feels out of place now.

Reply:
Why does it feel out of place?

Reply:
Let's leave it in then. 😉